test/shared/commonFunctionsForTests.sh

```diff
@@ -32,6 +32,7 @@ SED_CONFIG_3='s/end_date: \[20160201, 000000\]/end_date: \[20190101, 010000\]/'
 SED_CONFIG_4='s/end_date: \[20160101, 000000\]/end_date: \[20190101, 010000\]/'
 SED_CONFIG_5='s/end_date: \[20190201, 000000\]/end_date: \[20190101, 010000\]/'
 SED_CONFIG_6='s/end_date: \[20190801, 000000\]/end_date: \[20190701, 010000\]/'
+SED_CONFIG_7='s/end_date: \[20900801, 000000\]/end_date: \[20900701, 002000\]/'
```

Comment:
FYI, this has already gone into dev/no-diff-to-benchmark.  These updates will conflict with it.

Reply:
Approving with the understanding that @lizziel will rebase to dev/no-diff-to-benchmark.